app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkActivity.kt

```diff
@@ -181,9 +174,18 @@ class EmbarkActivity : BaseActivity(R.layout.activity_embark) {
         }
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        if (requestCode == REQUEST_MORE_OPTIONS && resultCode == RESULT_CANCELED) {
+            model.load(storyName)
```

Comment:
Should we flush the `store` at this point? 🤔

Reply:
Thought about that as well, I'll take it with Zak